### PR TITLE
Industrial Nail guns

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Added a new **Nail gun**, a .10 calibre device used to nail things in place and pin corpses to walls, all with a satisfying THUNK!
## Why / Balance
Do you need to arm the crew during zombies? Are you an aspiring agent for the syndicate? Is some asshole spilling your chem jugs? The Nail gun offers an answer to all of these!
## Media

TODO
- [ ] Finish the PR ((CHECK WHEN READY FOR MERGE))
- [x] Make handheld nail gun
- - [ ] Additional Nail guns
- - [ ] Test Nail gun functionality before merging ((CHECK WHEN READY FOR MERGE))
- [ ] Make nail gun Hackable
- - [ ] Make non-hacked nail gun
- [x] Nail boxes
- - [ ] Variant ammo
- - [ ] Re-sprite Nail box
- [ ] Nailing function ((CHECK WHEN READY FOR MERGE))
- - [ ] Nailing technology
- [ ] Restore mental wellbeing ((can be ignored))

- [ ] Ready for Review
- - [ ] Ready for Merging
## Breaking changes

**Changelog**
:cl:
add: Added Nail gun - Pin items down, or disable the safeties for an improvised weapon
add: Added new ammo, .10 cal Nails. Can be reused once more after being fired!

### CODE REVIEW

**Comment by UbaserB** [comment]
you messed up the CL

**Comment by JoeHammad1844** [comment]
Neat, i like the idea, but how will these be used outside of weapons?

**Comment by KittenColony** [comment]
By using this on an item, it will be "nailed down" which prevents you from picking it up, useful for things like jugs and important items, or anything you don't want moved, needing to be crowbarred to pry the nails out

It can also be used by cargo as an alternative to welding a crate shut

**Comment by asperger-sind** [comment]
> By using this on an item, it will be "nailed down" which prevents you from picking it up, useful for things like jugs and important items, or anything you don't want moved, needing to be crowbarred to pry the nails out

NAIL THE DISK

**Comment by KittenColony** [comment]
Yes.

**Comment by brainfood1183** [comment]
nail banana peels to floor

**Comment by MouseHatsForAll** [comment]
Seems balanced, the code isn't screaming at us that it's broken and there's sprites for it in-hand. Don't see why this couldn't be added.

**Comment by Kadeo64** [comment]
finally, my shitty concept idea of being able to anchor something and then nail it down to double anchor it is realized

**Comment by Hmeister-fake** [comment]
I can finally nail down Ian and other pets to their beds! Great addition.

**Comment by KittenColony** [comment]
As it stands this is very barely functional, please remember this

**Comment by KittenColony** [comment]
how did this piece of shit pass all the checks it crashes on roundstart

**Comment by deltanedas** [comment]
because its draft

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/nail.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     deleteOnSpawn: true
   - type: Sprite
     noRot: false
-    sprite: (nail sprite)
+    sprite: Objects\Weapons\Guns\Ammunition\Casings\nail.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/nail.yml:15]: \ pro use /

File: Resources/Audio/Weapons/Guns/Gunshots/attributions.yml [hunk 2]
```diff
@@ -21,4 +21,10 @@
...
   license: "CC-BY-SA-3.0"
   copyright: "Taken from tgstation"
-  source: "https://github.com/tgstation/tgstation/tree/88d7dbfc105fbf40284d7b7c4587f8d23c0ac3ac"
\ No newline at end of file
+  source: "https://github.com/tgstation/tgstation/tree/88d7dbfc105fbf40284d7b7c4587f8d23c0ac3ac"
+
+- files: ["nailgun.ogg]
+  license: "CC-BY-SA-3.0"
+  copyright: "Layering by KittenColony. Sounds from soundsnap & Rust"
+  source: "https://www.soundsnap.com/nailgun_01_wav, https://rust.facepunch.com"
```
Comment (deltanedas) [inline Resources/Audio/Weapons/Guns/Gunshots/attributions.yml:29]: since when is rust (game) cc by sa 3.0

Comment (KittenColony) [reply Resources/Audio/Weapons/Guns/Gunshots/attributions.yml:29]: > since when is rust (game) cc by sa 3.0

fuck

Comment (KittenColony) [reply Resources/Audio/Weapons/Guns/Gunshots/attributions.yml]: fixed

**Comment by Kadeo64** [comment]
really liking the trend of giving science more printable weaponry recently, I mean
-Crusher Glaive
-Svalinn Laser Pistol
-Nailgun (you are here)
-Foam Grenades (I mean, we already technically had this, but now we have stuff like razorium foam)

**Comment by Kadeo64** [comment]
Although, I have to ask, will this be a standard, research-required thing or an autolathe thing? I'm guessing autolathe.

**Comment by GoodWheatley** [comment]
> Although, I have to ask, will this be a standard, research-required thing or an autolathe thing? I'm guessing autolathe.

T2 Industrial

**Comment by Kadeo64** [comment]
> > Although, I have to ask, will this be a standard, research-required thing or an autolathe thing? I'm guessing autolathe.
> 
> T2 Industrial

With advanced tools or it's own separate thing? I would HIGHLY recommend putting it with advanced tools since Industrial Tier 2 is absurdly bloated. (I'm pretty sure industrial tier 2 has nearly as many unlocks as the entirety of biochemistry.)

**Comment by KittenColony** [comment]
what the fuck is it bitching about

**Comment by deltanedas** [comment]
GOD TIER use case: nailing artifact scans to the floor
1. shitters cant throw them everywhere
2. bozos that dont know how to read with verb cant pick it up
3. artifact magnet effect cant move them

**Comment by KittenColony** [comment]
what the fuck is breaking :(

**Comment by metalgearsloth** [comment]
@KittenColony  Is this still being worked on.

**Comment by KittenColony** [comment]
> @KittenColony Is this still being worked on.

yeah I just have a lot less time then I used to so I'm having to do this during lunch breaks and such instead of constantly

**Comment by KittenColony** [comment]
what the fuck

**Comment by KittenColony** [comment]
god fucking damn it

**Pull request closed without merging**